src/sentry/integrations/slack/webhooks/base.py

```diff
@@ -46,8 +46,8 @@ def post_dispatcher(self, request: SlackDMRequest) -> Response:
         """
         command, args = request.get_command_and_args()
 
-        if command in ["help", ""]:
-            return self.respond(SlackHelpMessageBuilder().build())
+        if command in ["help", "", "support", "docs"]:
```

Comment:
should we leave `""` currently if the user doesn't give us a command after `/sentry`, it will give the user the help response.

Reply:
oop i didn't read

Reply:
this still works even with "" https://github.com/getsentry/sentry/pull/73963/files#diff-cd09b7d4ee99699ebb00a4fa5f4fb54620e0591930926469c7b9a25572160eafR65-R68